nucypher/blockchain/economics.py

```diff
@@ -163,7 +164,8 @@ def staking_deployment_parameters(self) -> Tuple[int, ...]:
             # Constraints
             self.minimum_locked_periods,      # Min amount of periods during which tokens can be locked
             self.minimum_allowed_locked,      # Min amount of tokens that can be locked
-            self.maximum_allowed_locked       # Max amount of tokens that can be locked
+            self.maximum_allowed_locked,      # Max amount of tokens that can be locked
+            self.minimum_worker_periods       # Min amount of periods while a worker can't be changed
```

Comment:
"Minimum number of periods a contract can be an intermediary "

Reply:
Did not get what do you mean,
this value to prevent too often changing of worker, but staker is always an intermediary contract or real address